scanner/updater/export.go

```diff
@@ -66,11 +67,25 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 		bundles[uSet] = []updates.ManagerOption{updates.WithEnabled([]string{uSet})}
 	}
 
+	// Rate limit to ~16 requests/second by default.
+	interval := 62*time.Millisecond
+	configuredInterval := os.Getenv("STACKROX_SCANNER_V4_UPDATER_INTERVAL")
+	if configuredInterval != "" {
+		parsedInterval, err := time.ParseDuration(configuredInterval)
```

Comment:
Late nit - may be able to simplify this by using `pkg/env` `DurationSetting`

Reply:
I forgot about that 🤦

Reply:
eh actually `reigsterSetting` is private, and I don't want to modify that file in this PR. I'll just leave it as-is